thunder/tests/test_jit_general.py

```diff
@@ -787,6 +787,9 @@ def test_tom_overrides_proxy(device):
     ("cpu", "cuda"),
 )
 def test_cache_symbolic_values(device):
+    if device == "cuda" and not torch.cuda.is_available():
```

Comment:
nit: i doubt it matters, but might consider testing if "cuda" is a substr of device or something, so that e.g. `cuda:0` still does the right thing

Reply:
but this is from the parametrize in 787...

Reply:
That's fair. though we did provide the device string in the parameter above. :) 
`    ("cpu", "cuda"),`